### PR TITLE
UseBasicParsing on web requests in Powershell

### PR DESCRIPTION
**Pull request by scottbommarito**
If web requests are made without this attribute, they will fail on machines that have not completed first-run configuration of Internet Explorer (VSTS hosted build agents).

### CODE REVIEW

**Comment by joelverhagen** [comment]
Can you do a build locally using `build.ps1` and verify that it doesn't create any pending changes?

File: build.ps1 [hunk 1]
```diff
@@ -11,7 +11,7 @@ param (
     [string]$SemanticVersion = '1.0.0-zlocal',
     [string]$Branch,
     [string]$CommitSHA,
-    [string]$BuildBranch = 'bff597e5990a14bb6ca53e371313d54bec48b4a6'
+    [string]$BuildBranch = '0bedf795cde346cc55f9aa5199914cafd6a4d15a'
```
Comment (cristinamanum) [inline build.ps1]: why is this id changed?

Comment (scottbommarito) [inline build.ps1]: Corresponds to a commit in ServerCommon that holds some utility scripts that this uses to run, for example https://github.com/NuGet/ServerCommon/blob/master/build/common.ps1.

**Comment by scottbommarito** [comment]
@joelverhagen Yes.

**Pull request merged**